IDLE_DWELL/Find_Idle_Dwells.py

```diff
@@ -0,0 +1,256 @@
+    else:
+        # The review load is a TOO, SCS-107 or Full stop Return to Science load
+        # So read the review load's ACIS-History_edit.txt file
+        hist_dict = oflsfu.Read_ACIS_History_file(load_week_path, edit=True)
+
+    # Now set the value of previous and present instrument
+    # from the hist dictionary
+    state_dict["present_fp_instrument"] = hist_dict["instrume"]
+    state_dict["previous_fp_instrument"] = hist_dict["instrume"]
+
+    # Capture the RADMON status and store in the state dictionary
+    state_dict["radmon_state"] = hist_dict["radmon_status"]
+
+    # Find and read the CR*.backstop file
+    # First formulate the path to the CR*.backstop file with wild cards
+    partial_path = "/".join((load_week_path, "CR*.backstop"))
+
+    # Get the name of the CR*.backstop file. There is only one per OFLS directory
+    backstop_file_path = glob.glob(partial_path)[0]
+
+    # Create an instance of the Backstop File Processing Class
+    BFCI = bfp.Backstop_File_Class()
+    
+    # Read the backstop file and extract all commands of command type:
+    #    ACISPKT, SIMTRANS,  OORMPDS, OORMPEN
+
+    all_commands = BFCI.Read_BS_File(backstop_file_path) 
+
+    # Extract the SIM, ACISPKT and RADMON commands out of all of the commands
+    sim_acis_commands = BFCI.Extract_Type_and_TLMSID(["SIMTRANS", "ACISPKT", "OORMPDS", "OORMPEN"], all_commands)
+
+    # ----------------------------------  BEGIN DWELL PROCESSING ---------------------------------------------------
+    # The assumption here is that at the start of a load, you will not be in the middle of
+    # any kind of observation (neither science nor ECS)
+    # So for the first check, set the start of the search be the first command,
+    # and look for the first Start Science.
+    
+    dwell_start_date = all_commands[0]["date"]
+    dwell_start_time = all_commands[0]["time"]
+    
+    # Now scan through all the rest of the extracted  commands until
+    # you see a start science. That defines the first "dwell"
+    index = 0
+
+    while index < len(sim_acis_commands):
+
+        # Is this an OORMPDS command? If so set the value of the
+        # state dictionary
+         if "OORMPDS" in  sim_acis_commands[index]["tlmsid_string"]:
+             state_dict["radmon_status"] = "RADMON_DIS"
+                   
+        # Is this an OORMPEN command? If so set the value of the
+        # state dictionary
+         if "OORMPEN" in  sim_acis_commands[index]["tlmsid_string"]:
+             state_dict["radmon_status"] = "RADMON_EN"
+        
+        # Is this a SIMTRANS? If so set the present instrument to the commanded value
+         if sim_acis_commands[index]["command_type"] == "SIMTRANS":
+            # If was_an_obs is False, set the previous FP instrument to the
+            # present FP instrument. When there was a previous observation,
+            # the processing for the AA00 command does this.
+            if state_dict["was_an_obs"] == False:
```

Comment:
Nit, but suggest `if not state_dict["was_an_obs"]:`

Reply:
Don't agree with this one because "not" doesn't make it clear that state_dict["was_an_obs"] is a boolean. 

if you had x = [] then not x would also resolve to True.

the use of False reminds the reader at this point that state_dict["was_an_obs"] is a boolean